### ISSUE TITLE
TIFF stack with no scale?

### ISSUE THREAD

**Issue by tjof2** [hyperspy/hyperspy#1167]
I've got a TIFF stack (256x256x103) with no scale set - this is what it looks like when I load it into FIJI:

![gimp](https://cloud.githubusercontent.com/assets/6626247/16919522/ef51afce-4d01-11e6-8745-091739202d68.png)

Trying to load it with HyperSpy throws this error:

```
---------------------------------------------------------------------------
IndexError                                Traceback (most recent call last)
<ipython-input-4-f56864702d12> in <module>()
----> 1 cs_si = hs.load('stack.tif')
      2 cs_si.change_dtype('float')
      3 X = cs_si.data.transpose(1,2,0)
      4 m, n, t = X.shape
      5 X = X.reshape(m*n, t)

/home/tjof2/hyperspy/hyperspy/io.py in load(filenames, signal_type, stack, stack_axis, new_axis_name, mmap, mmap_dir, **kwds)
    177             objects = [load_single_file(filename,
    178                                         **kwds)
--> 179                        for filename in filenames]
    180 
    181         if hyperspy.defaults_parser.preferences.Plot.plot_on_load:

/home/tjof2/hyperspy/hyperspy/io.py in <listcomp>(.0)
    177             objects = [load_single_file(filename,
    178                                         **kwds)
--> 179                        for filename in filenames]
    180 
    181         if hyperspy.defaults_parser.preferences.Plot.plot_on_load:

/home/tjof2/hyperspy/hyperspy/io.py in load_single_file(filename, signal_type, **kwds)
    221                                 reader=reader,
    222                                 signal_type=signal_type,
--> 223                                 **kwds)
    224 
    225 

/home/tjof2/hyperspy/hyperspy/io.py in load_with_reader(filename, reader, signal_type, **kwds)
    229                      **kwds):
    230     file_data_list = reader.file_reader(filename,
--> 231                                         **kwds)
    232     objects = []
    233 

/home/tjof2/hyperspy/hyperspy/io_plugins/tiff.py in file_reader(filename, record_by, **kwds)
    183                 _logger.info("Reading ImageJ tif metadata")
    184                 # ImageJ write the unit in the image description
--> 185                 units = image_description.split('unit=')[1].split('\n')[0]
    186                 scales = _get_scales_from_x_y_resolution(op)
    187 

IndexError: list index out of range
```

**Comment by tjof2** [comment]
I seem to have the same problem if "unit" is set to "pixel" in FIJI. Only by specifying "nm" was I able to get it to load.

**Comment by francisco-dlp** [comment]
Fixed in #1167
